src/meta/src/model/migration_plan.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use prost::Message;
+use risingwave_pb::common::ParallelUnit;
+use risingwave_pb::meta::PbMigrationPlan;
+
+use crate::storage::{MetaStore, MetaStoreError, MetaStoreResult, Snapshot, DEFAULT_COLUMN_FAMILY};
+
+/// `migration_plan` in bytes.
+pub const MIGRATION_PLAN_KEY: &[u8] = &[
+    109, 105, 103, 114, 97, 116, 105, 111, 110, 95, 112, 108, 97, 110,
+];
+
+type WorkerId = u32;
+type ParallelUnitId = u32;
+
+#[derive(Debug, Default, Clone)]
+pub struct MigrationPlan {
```

Comment:
Why don't we impl MetadataModel any longer?

Reply:
It's a simple raw key-value pair in meta store, which doesn't fit the definition of the model somehow. So I prefer to implement it like some other kv pairs in meta that we have maintained in default cf.